packages/hash/frontend/src/pages/[account-slug]/types/entity-type/edit-bar.tsx

```diff
@@ -0,0 +1,110 @@
+import { faSmile } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon } from "@hashintel/hash-design-system/fontawesome-icon";
+import { Box, Container, Stack, Typography } from "@mui/material";
+import { ReactNode } from "react";
+import { PencilSimpleLine } from "../../../../shared/icons/svg";
+import { Button, ButtonProps } from "../../../../shared/ui/button";
+
+const EditBarContents = ({
+  icon,
+  title,
+  label,
+  discardButtonProps,
+  confirmButtonProps,
+}: {
+  icon: ReactNode;
+  title: ReactNode;
+  label: ReactNode;
+  discardButtonProps: ButtonProps;
+  confirmButtonProps: ButtonProps;
+}) => (
+  <Container
```

Comment:
Optional, for all similar cases: it'd useful to use explicit `return` statements in all React components instead of this arrow function shortcut. If we need to add some hook later, we won't end up with a crazy diff.

Reply:
Is there an autofixable lint rule for this? I tend to agree (from an "optimise for change" perspective) – although not for diff reasons because you can just hide whitespace in the diff, just for reducing friction of adding a hook. You could create a ticket?

Reply:
> you can just hide whitespace in the diff

That won't help if JSX is wide enough to be re-formatted. 

I created https://app.asana.com/0/1200211978612931/1203132574912430/f